merge_command.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+import "strings"
+
+func isMergeCommand(comment string) bool {
+	return strings.TrimSpace(comment) == "!merge"
```

Comment:
Minor style comment - in operations like this, I like to put the value first:
`"!merge" == strings.TrimSpace(comment)`
Places emphasis on the value.